webapp/src/components/App/Studies/StudiesList/index.tsx

```diff
@@ -91,6 +94,7 @@ function StudiesList(props: StudiesListProps) {
   const [selectedStudies, setSelectedStudies] = useState<string[]>([]);
   const [selectionMode, setSelectionMode] = useState(false);
   const [confirmFolderScan, setConfirmFolderScan] = useState<boolean>(false);
+  const [requestDeepScan, setRequestDeepScan] = useState<boolean>(false);
```

Comment:
Rename state variable for clarity:
- `requestDeepScan` -> `isRecursiveScan` (prefix "is" indicating a boolean flag)

Matches the API parameter name in `scanFolder()`

Also when you add a default state you dont have to specify the type, TS will infer it
`const [isRecursiveScan, setIsRecursiveScan] = useState(false);`